cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -950,22 +974,7 @@ public void testExposesJmxContainerPortWhenJmxEnabled() {
         }));
     }
 
-    @ParallelTest
-    public void testContainerPorts() {
-        Kafka kafka = new KafkaBuilder(KAFKA)
-                .editSpec()
-                    .editKafka()
-                        .withListeners(new GenericKafkaListenerBuilder().withName("tls").withPort(9093).withType(KafkaListenerType.INTERNAL).withTls().build(),
-                                new GenericKafkaListenerBuilder().withName("external").withPort(9094).withType(KafkaListenerType.NODEPORT).withTls().build())
-                        .withNewJmxPrometheusExporterMetricsConfig()
-                            .withNewValueFrom()
-                                .withNewConfigMapKeyRef("metrics-cm", "metrics.json", false)
-                            .endValueFrom()
-                        .endJmxPrometheusExporterMetricsConfig()
-                    .endKafka()
-                .endSpec()
-                .build();
-
+    private void testContainerPorts(Kafka kafka) {
```

Comment:
Rename to `assertExpectedContainerPortsAreSet`, start from `podSets.stream().forEach...`, and move right after the calling test methods.

Reply:
Is this what you mean?